### PR TITLE
RTN15d

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

**Comment by ricardopereira** [comment]
I think this should be rebased with #231 fixes.

**Comment by ricardopereira** [comment]
I think this should be rebased with #228 fixes.

File: ablySpec/RealtimeClientConnection.swift [hunk 1]
```diff
@@ -1302,6 +1302,54 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN15
+            context("connection failures once CONNECTED") {
+
+                // RTN15d
+                pending("should recover from disconnection and messages delivered once the connection is resumed") {
```
Comment (mattheworiordan) [inline ablySpec/RealtimeClientConnection.swift]: Can we change this to "should recover from disconnection and publish queued messages once the connection is resumed"?

Comment (mattheworiordan) [inline ablySpec/RealtimeClientConnection.swift]: Actually, I may have misread RTN15d and this suggested change is wrong. However, is this test then not the same as RTN15a, I'm not sure why there are two different tests?

Comment (ricardopereira) [inline ablySpec/RealtimeClientConnection.swift]: You right, I noticed [that](https://github.com/ably/ably-ios/pull/244#issuecomment-187715273). I will change the #244.

**Comment by ricardopereira** [comment]
Rebased with master with changes: 7b8e8f5.

**Comment by mattheworiordan** [comment]
LGTM

**Comment by tcard** [comment]
LGTM

**Pull request merged**